virt/virt-learn-more-about-openshift-virtualization.adoc

```diff
@@ -1,109 +1,66 @@
+
+[NOTE]
+====
+Cluster configuration procedures require `cluster-admin` privileges.
+====
+
+[id="virt-getting-started-before-you-begin"]
+== Before you begin
+
+* Review the xref:../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[installation requirements].
+* Review the xref:../virt/virtual_machines/virtual_disks/virt-features-for-storage.adoc#virt-features-for-storage[storage features] required for cloning, snapshots, and live migration. See xref:../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[Using a CSI-enabled storage provider] for details.
+* Install the xref:../virt/install/installing-virt-web.adoc#virt-installing-virt-operator_installing-virt-web[{VirtProductName} Operator] and the xref:../virt/install/virt-enabling-virtctl.adoc#virt-enabling-virtctl[`virtctl` tool].
+
+[id="virt-getting-started"]
+== Getting started
+
+Create a virtual machine:
+
+* Create a xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-wizard-web_virt-create-vms[RHEL virtual machine] using the wizard.
+* Create a Windows virtual machine:
+** Create and customize link:https://cloud.redhat.com/blog/virtual-machines-as-code-with-openshift-gitops-and-openshift-virtualization[Windows boot sources]
+** Create a xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-wizard-web_virt-create-vms[Windows virtual machine] using the wizard.
+** Install xref:../virt/virtual_machines/virt-installing-qemu-guest-agent.adoc#virt-installing-virtio-drivers-installing-windows_virt-installing-qemu-guest-agent[VirtIO drivers] on the virtual machine.
+
+Connect to a virtual machine:
+
+* Connect to a virtual machine using xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-connecting-to-the-terminal_virt-accessing-vm-consoles[the terminal] or xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-vm-serial-console-web_virt-accessing-vm-consoles[the serial console] in the {product-title} web console.
+* Connect to a virtual machine using SSH:
+** xref:../virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc#virt-creating-a-service-from-a-virtual-machine_virt-using-the-default-pod-network-with-virt[Create a service] to expose a route for the virtual machine.
+** Use the service to connect to the virtual machine with xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-accessing-vmi-ssh_virt-accessing-vm-consoles[SSH].
+* xref:../virt/virtual_machines/virt-controlling-vm-states.adoc#virt-controlling-vm-states[Stop, start, pause, and restart] the virtual machine.
+
+[id="virt-getting-started-next-steps"]
+== Next steps
+
+Configure networks:
+
+* Configure the virtual machine for the xref:../virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc#virt-using-the-default-pod-network-with-virt[default pod network].
```

Comment:
Something like this would be more precise:
```suggestion
* VMs created from the included templates are using the xref:../virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc#virt-using-the-default-pod-network-with-virt[default pod network].
```

Reply:
I wonder whether we even need this step. If the user has created VMs from templates, they do not need to configure the VMs for the default pod network. It might be better to remove this step.

Reply:
Cool. Thanks! Done.